.storybook/preview.tsx

```diff
@@ -1,12 +1,13 @@
+import "./styles.css";
+
 import { DARK_MODE_EVENT_NAME } from "storybook-dark-mode";
 import { DocsContainer } from "@storybook/addon-docs";
 import React from "react";
 import { addons } from "@storybook/preview-api";
+import { store } from "storybook-dark-mode/src/Tool";
```

Comment:
Reusing some of the internal implementation of storybook-dark-mode is not ideal as it is prone to break at any time. That said, they don't expose a way to read from localstorage to initialise state like they do internally. This seems like the best bad option to avoid flickering a fallback theme.